Core Modules/WalletConnectSharp.Network.Websocket/WebsocketConnection.cs

```diff
@@ -74,7 +74,14 @@ public EventDelegator Events
                 return _delegator;
             }
         }
-        
+
+        public event EventHandler<string> PayloadReceived;
+        public event EventHandler Closed;
+        public event EventHandler<DisconnectionInfo> WebsocketClosed;
```

Comment:
This event is invoked together with `Closed`, but has no subscriptions and isn't a member of `IJsonRpcConnection`. Why not remove it, and add `DisconnectionInfo` arg to `Closed`?